scripts/globals/spells/enfeebling_spell.lua

```diff
@@ -16,75 +16,75 @@ xi.spells.enfeebling = xi.spells.enfeebling or {}
-    [xi.magic.spell.INUNDATION   ] = { xi.effect.INUNDATION,         xi.mod.MND,     0,                  0,                       1,   0,      300,      5,   0,        0, false,     false   },
-    [xi.magic.spell.PARALYZE     ] = { xi.effect.PARALYSIS,          xi.mod.MND,     xi.mod.PARALYZERES, xi.mod.PARALYZE_MEVA,    0,   0,      120,      2,   0,       32, true,      true    },
-    [xi.magic.spell.PARALYZE_II  ] = { xi.effect.PARALYSIS,          xi.mod.MND,     xi.mod.PARALYZERES, xi.mod.PARALYZE_MEVA,    0,   0,      120,      2,   0,       32, true,      true    },
-    [xi.magic.spell.PARALYGA     ] = { xi.effect.PARALYSIS,          xi.mod.MND,     xi.mod.PARALYZERES, xi.mod.PARALYZE_MEVA,    0,   0,      120,      2,   0,       32, true,      true    },
-    [xi.magic.spell.REPOSE       ] = { xi.effect.SLEEP_II,           xi.mod.MND,     xi.mod.SLEEPRES,    xi.mod.SLEEP_MEVA,       2,   0,       90,      2,   1,        1, false,     false   },
-    [xi.magic.spell.SILENCE      ] = { xi.effect.SILENCE,            xi.mod.MND,     xi.mod.SILENCERES,  xi.mod.SILENCE_MEVA,     1,   0,      120,      2,   0,       16, false,     false   },
-    [xi.magic.spell.SILENCEGA    ] = { xi.effect.SILENCE,            xi.mod.MND,     xi.mod.SILENCERES,  xi.mod.SILENCE_MEVA,     1,   0,      120,      2,   0,       16, false,     false   },
-    [xi.magic.spell.SLOW         ] = { xi.effect.SLOW,               xi.mod.MND,     xi.mod.SLOWRES,     xi.mod.SLOW_MEVA,        0,   0,      180,      2,   0,      128, true,      true    },
-    [xi.magic.spell.SLOW_II      ] = { xi.effect.SLOW,               xi.mod.MND,     xi.mod.SLOWRES,     xi.mod.SLOW_MEVA,        0,   0,      180,      2,   0,      128, true,      true    },
-    [xi.magic.spell.SLOWGA       ] = { xi.effect.SLOW,               xi.mod.MND,     xi.mod.SLOWRES,     xi.mod.SLOW_MEVA,        0,   0,      180,      2,   0,      128, true,      true    },
+    [xi.magic.spell.FLASH        ] = { xi.effect.FLASH,              xi.mod.MND,     xi.mod.BLINDRES,    xi.mod.BLIND_MEVA,     300,   0,       12,      4,   0,        0, true,      false,   200 },
+    [xi.magic.spell.INUNDATION   ] = { xi.effect.INUNDATION,         xi.mod.MND,     0,                  0,                       1,   0,      300,      5,   0,        0, false,     false,     0 },
+    [xi.magic.spell.PARALYZE     ] = { xi.effect.PARALYSIS,          xi.mod.MND,     xi.mod.PARALYZERES, xi.mod.PARALYZE_MEVA,    0,   0,      120,      2,   0,       32, true,      true ,   -10 },
+    [xi.magic.spell.PARALYZE_II  ] = { xi.effect.PARALYSIS,          xi.mod.MND,     xi.mod.PARALYZERES, xi.mod.PARALYZE_MEVA,    0,   0,      120,      2,   0,       32, true,      true ,     0 },
+    [xi.magic.spell.PARALYGA     ] = { xi.effect.PARALYSIS,          xi.mod.MND,     xi.mod.PARALYZERES, xi.mod.PARALYZE_MEVA,    0,   0,      120,      2,   0,       32, true,      true ,     0 },
```

Comment:
paralyze II and paralyga should have the same penalty as paralyze I, right?
slow and slow II had the same bonus after all

Reply:
These are the only data points that I have at the moment. It is very likely that the Tier 1s either have 0 or less macc than tier 2 and so on.

This was added by SE so that the higher the tier the better the magic accuracy would be - this is the only data I have at the moment but this should make it easier to add in the future when more data has been gathered.

Reply:
Will have to do for the time being. Good job, man